workspaces/lluis/Tag.svelte

```diff
@@ -0,0 +1,21 @@
+<script lang="typescript">
+  export let color = null
+  export let theme = null
+  export let size = null
+  export let isDelete = false
+  export let isRounded = false
+  export let style = ""
+</script>
+
+<span {style} class="tag {`tag ${size ? `is-${size}`: ''} ${theme ? `is-${theme}`: ''} ${color ? `is-${color}`: ''}`}"
+class:is-rounded={isRounded}
+class:is-delete={isDelete}
+>
+  {#if !isDelete}
```

Comment:
hmm, I'm not sure why this `isDelete` is necessary here :thinking: I think we're not really using this, no?

You can freely delete anything that we're not really using. Lets not maintain unused code :-)